README.md

```diff
@@ -50,15 +50,25 @@ To clone the repository, complete the following steps:
 
 ## Get started
 
-For an overview of the user interface and features, refer to
-the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
-
-To review code samples and start building, refer to
-the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
-Codelab.
-
-To make sure that your device has the latest Matter support, review the
-[Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services) guide.
+*   For an overview of the user interface and features, refer to
+    the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
+*   To review code samples and start building, refer to
+    the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
+    Codelab.
+*   To make sure that your device has the latest Matter support, review the
+    [Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services)
+    guide.
+*   [Create a Matter integration](https://developers.home.google.com/matter/integration/create) and
+    build a Matter device with On/Off capabilities. This sample app works with a virtual device
+    and an ESP32.
+    *   [Build a Matter Virtual Device](https://developers.home.google.com/codelabs/matter-device-virtual)
+        with the `rootnode_dimmablelight_bCwGYSDpoe` app. For **Create new project** >
+        **Configuration** in the [Developer Console](https://console.home.google.com/projects),
```

Comment:
The console guidance of "Create new project" seems out of sync with the "Create a Matter integration" link above, in which you should've already created a project.  Same with the other below.